docs/integrations/sources/google-search-console.md

```diff
@@ -82,6 +80,15 @@ If you lose this key pair, you will need to generate a new one!
 ### Note
 You can return to the [API Console/Credentials](https://console.cloud.google.com/apis/credentials) at any time to view the email address, public key fingerprints, and other information, or to generate additional public/private key pairs. For more details about service account credentials in the API Console, see [Service accounts](https://cloud.google.com/iam/docs/understanding-service-accounts) in the API Console help file.
 
+### Using the existing User Account 
```

Comment:
no need to add this section to the external docs as it will be automated via oauth in the UI. This is an excellent README however to place in the `credentials/` directory.